flight/mcu/lib/teensyShared/kalmanFilter.h

```diff
@@ -13,8 +13,10 @@ class KalmanFilter {
     void Initialize(float pos_f, float vel_f);
     void priori();
```

Comment:
I've been curious for a while why this function is named in latin

Reply:
so the Kalman filter fwiw has 2 steps, the priori and the update. In this case, an "a priori" estimate is your estimate of where you are since you don't have a current sensor measurement. It's the same as the priori and posteriori in Bayesian probability since you are making some extrapolation given knowledge about the past.

Reply:
Yeah I can get that done